examples/ui/z_index.rs

```diff
@@ -0,0 +1,223 @@
+//! Demonstrates how to use z-index
+//! Shows two colored buttons with transparent text.
```

Comment:
Is this correct?

In the video of the PR description there are a lot more elements than two and no transparent text.

Reply:
This is indeed a mistake because the example was improved. I will edit the description.